Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -86,16 +86,25 @@ extension LottieLoopMode: Equatable {
 
 // MARK: - AnimationView
 
+@available(
+  *,
+  deprecated,
+  renamed: "LottieAnimationView",
+  message: "`AnimationView` has been renamed to `LottieAnimationView`, for consistency with the new `LottieAnimation` type.")
+public typealias AnimationView = LottieAnimationView
+
+// MARK: - LottieAnimationView
+
 @IBDesignable
-final public class AnimationView: AnimationViewBase {
+final public class LottieAnimationView: LottieAnimationViewBase {
```

Comment:
Have you put any thoughts into what we'd name a SwiftUI `View` for Lottie animations? If both `LottieAnimation` and `LottieAnimationView` are taken I wonder what options we would have

Reply:
Good question, `LottieView` is a potential option. How have we been handling this in other cases, where we provide a UIView and SwiftUI View with the same functionality?

Reply:
Sure, I think `LottieView` would work!